GDJS/Runtime/pixi-renderers/pixi-filters-tools.js

```diff
@@ -96,6 +96,43 @@ gdjs.PixiFiltersTools._filters = {
             filter.alpha = value;
         },
     },
+    BlackAndWhite: {
+        makeFilter: function() {
+            var colorMatrix = new PIXI.filters.ColorMatrixFilter();
+            colorMatrix.blackAndWhite();
+            return colorMatrix;
+        },
+        updateParameter: function(filter, parameterName, value) {
+            if (parameterName !== 'opacity') return;
+
+            filter.alpha = value;
+        },
+    },
+    Noise: {
+        makeFilter: function() {
+            var noise = new PIXI.filters.NoiseFilter();
+            return noise;
+        },
+        updateParameter: function(filter, parameterName, value) {
+            if (parameterName !== 'noise') return;
+
+            filter.noise = value;
+        },
+    },
+    Blur: {
+        makeFilter: function() {
+            var blur = new PIXI.filters.BlurFilter();
+            return blur;
+        },
+        updateParameter: function(filter, parameterName, value) {
+            if (parameterName !== 'blur' &&
```

Comment:
While it's not done for existing effects, I think it would be worth adding a bit of extra check to avoid invalid values to be set for the parameters. What do you think? For example, return if the kernelSize is not a valid value, or value is out of range, etc.

Reply:
Yep makes sense.
I have already implemented a check for the kernelSize in [EffectDescription.js](https://github.com/4ian/GDevelop/pull/1206/commits/c1da506b419e862c4fcbfdca0693b9d3629df41e#diff-6c71ccf608de8a3035773caafdc2230aR15) 
Moved the checks to `pixi-filters-tools.js` so that we don't have to declare the same check functions twice.